spec/integration/apply_compile_spec.rb

```diff
@@ -191,6 +191,21 @@ def get_notifies(result)
           'hiera-config' => File.join(__dir__, '../fixtures/apply/hiera_invalid.yaml').to_s
         }
       }
+      let(:eyaml_config) {
```

Comment:
I could not figure out how to reliably add a relative path in a hiera config file, so instead of committing one like the other tests, i'm constructing one dynamically. Specifically, the paths to the `pkcs7` key pair.